src/hooks.server.ts

```diff
@@ -0,0 +1,28 @@
+import { appendCookieHeader, JWT_ACCESS_KEY, JWT_REFRESH_KEY } from '$lib/auth';
+import type { Handle } from '@sveltejs/kit';
+
+async function _handleAuthentication(apiResponse: Response): Promise<Response> {
+  if (!apiResponse.ok) {
+    return apiResponse;
+  }
+
+  const serverResponse = new Response(apiResponse.clone().body, apiResponse);
+  const json = await apiResponse.json();
+
+  const accessToken = json[JWT_ACCESS_KEY];
+  if (accessToken) {
+    appendCookieHeader(serverResponse, JWT_ACCESS_KEY, accessToken);
+  }
+
+  const refreshToken = json[JWT_REFRESH_KEY];
+  if (refreshToken) {
+    appendCookieHeader(serverResponse, JWT_REFRESH_KEY, refreshToken);
+  }
+
+  return serverResponse;
+}
+
+export const handle: Handle = async function ({ event, resolve }) {
```

Comment:
```suggestion
export async function handle({ event, resolve }): Handle {
```
Any reason to not use this syntax or an arrow function? Assigning a variable like this seems odd

Reply:
Wouldn't that imply that the function returns Handle? In my implementation, Handle is the type of the function

Reply:
Ohh that makes sense, never mind then. The docs uses the `satisfies` but it should be the same